pandas-stubs/core/indexes/timedeltas.pyi

```diff
@@ -42,5 +50,10 @@ class TimedeltaIndex(DatetimeTimedeltaMixin, TimedeltaIndexProperties):
     def to_series(self, index=..., name=...) -> TimedeltaSeries: ...
 
 def timedelta_range(
-    start=..., end=..., periods=..., freq=..., name=..., closed=...
+    start: TimedeltaConvertibleTypes = ...,
+    end: TimedeltaConvertibleTypes = ...,
```

Comment:
`start` and `end` can be `"1 day"`, for example, so since the quantity can be anything, both of these should be `str`

Reply:
`str` is a member of TimedeltaConvertibleTypes.

Reply:
All of the convertible types are tested.